packages/hash/api/src/model/knowledge/user.model.ts

```diff
@@ -200,7 +200,7 @@ export default class extends EntityModel {
     const entityTypeModel = SYSTEM_TYPES.entityType.user;
 
     const entity = await EntityModel.create(graphApi, {
-      ownedById: systemAccountId,
+      ownedById: systemOrgAccountId,
```

Comment:
See comment above about org entity ownership

Reply:
This is now consistent with what we're doing in `org.model.ts` after the change in https://github.com/hashintel/hash/commit/5a6fc01c844c537a3a4c3ba30c9f6d903aa0fbc0